operations/app/src/modules/function_app/main.tf

```diff
@@ -1,3 +1,43 @@
+locals {
+  all_app_settings = {
+    "POSTGRES_USER" = "${data.azurerm_key_vault_secret.postgres_user.value}@${data.azurerm_postgresql_server.postgres_server.name}"
+    "POSTGRES_PASSWORD" = data.azurerm_key_vault_secret.postgres_pass.value
+
+    "PRIME_ENVIRONMENT" = (var.environment == "prod" ? "prod" : "test")
+
+    "OKTA_baseUrl" = "hhs-prime.okta.com"
+    "OKTA_redirect" = var.okta_redirect_url
+
+    # Manage client secrets via a Key Vault
+    "CREDENTIAL_STORAGE_METHOD" ="AZURE"
+    "CREDENTIAL_KEY_VAULT_NAME" = "${var.resource_prefix}-clientconfig"
+
+    # Manage app secrets via a Key Vault
+    "SECRET_STORAGE_METHOD" = "AZURE"
+    "SECRET_KEY_VAULT_NAME" = "${var.resource_prefix}-appconfig"
+
+    # Route outbound traffic through the VNET
+    "WEBSITE_VNET_ROUTE_ALL" = 1
+
+    # Route storage account access through the VNET
+    "WEBSITE_CONTENTOVERVNET" = 1
+
+    # Use the VNET DNS server (so we receive private endpoint URLs
+    "WEBSITE_DNS_SERVER" = "168.63.129.16"
+
+    "DOCKER_REGISTRY_SERVER_URL" = data.azurerm_container_registry.container_registry.login_server
+    "DOCKER_REGISTRY_SERVER_USERNAME" = data.azurerm_container_registry.container_registry.admin_username
+    "DOCKER_REGISTRY_SERVER_PASSWORD" = data.azurerm_container_registry.container_registry.admin_password
+    "DOCKER_CUSTOM_IMAGE_NAME" = "${data.azurerm_container_registry.container_registry.login_server}/${var.resource_prefix}:latest"
```

Comment:
Latest will be deployed when starting from scratch, then the version tag will be used in the future via the GitHub Action.